src/lib/io/include/sol-iio.h

```diff
@@ -77,6 +77,7 @@ typedef struct sol_iio_config {
     const void *data; /**< User defined data to be sent to sol_iio_reader_cb */
```

Comment:
Thanks @edersondisouza, I increased the version to 2, please check if there is missing